### PR TITLE
Fixed Admin responsive views

### PR DESCRIPTION
**Pull request by srestrep74**
# Description

I fix the aside menu of the admin views so this can be responsive, and also de width of the data tables.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by mahoyos — COMMENTED** [review]
Put all the text in the LANG file

File: resources/views/admin/order.blade.php
Comment (LuisaMG01) [inline resources/views/admin/order.blade.php:1]: the more details code is not in lang

File: resources/views/partials/admin/adminNavbar.blade.php
Comment (LuisaMG01) [inline resources/views/partials/admin/adminNavbar.blade.php:1]: the open sidebar text should be in lang

**Review by LuisaMG01 — COMMENTED** [review]
There are two text into the code that sould be in lang

**Review by JuanM0412 — APPROVED** [review]

**Pull request closed without merging**